README.md

```diff
@@ -34,9 +34,15 @@ A bit more about the different fields...
 - `allow_private` (optional)
     - bool [true, false]
     - default: false
-    - If false bugs that are not public will not be synchronized. Note that in order to synchronize
+    - If `false` bugs that are not public will not be synchronized. Note that in order to synchronize
       private bugs the bugzilla user that JBI runs as must be in the security groups that are making
       the bug private.
+- `brackets_required` (optional)
```

Comment:
If this option really necessary? I ask because no configuration entry was modified in this PR